crates/superposition_types/migrations/2025-02-11-100518_remove_description_config_version/up.sql

```diff
@@ -1,3 +1,3 @@
 -- Your SQL goes here
 ALTER TABLE public.config_versions
-DROP COLUMN description;
\ No newline at end of file
+DROP COLUMN change_reason;
```

Comment:
@Datron change is correct , 
can we rename migration file from remove_description -> remove_change_reason